Samples/Windowing/cs-wpf/wpf_packaged_app/wpf_packaged_app.csproj

```diff
@@ -18,9 +18,15 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Content Update="Assets\window_icon.png">
+    <Content Update="window_icon.png">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
   </ItemGroup>
 
+  <ItemGroup>
+    <None Update="window_icon.png">
```

Comment:
Do we need both `ItemGroup`s?

Reply:
@mevey Do you know the answer to this? I am not sure if we need both None and Content item groups

Reply:
Going with the `None` item group since we don't need to include in publish